pkgs/development/python-modules/expecttest/default.nix

```diff
@@ -0,0 +1,28 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, poetry
+}:
+
+buildPythonPackage rec {
+  pname = "expecttest";
+  version = "0.1.3";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "ezyang";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-5CnpVFSbf3FcAa06Y7atG8sxu8uevpfrliB2HuVcrx0=";
+  };
+
+  buildInputs = [ poetry ];
+
```

Comment:
Missing pythonImportsCheck.
Can we run https://github.com/ezyang/expecttest/blob/main/test_expecttest.py ?

Reply:
Oh good catch, fixed in https://github.com/NixOS/nixpkgs/pull/174248/commits/4edbc05a5eb5124822bd2066f85ec5abfa452f54